### DISCUSSION TITLE
Issue with Pro and Lombok ?

### DISCUSSION THREAD

**Discussion by jhanzel-be** [jobrunr/jobrunr#702] [General]
Hi, 

Our team is in the process of upgrading to JobRunrPro, we have been using the free version for some time very successfully.   Unfortunately, we are running into an issue right out of the gate.  After updating our pom and attempting to compile our project with the Pro version, we are getting lots (100's) of compiler issues related to fields and methods that are generated using lombok. The errors indicate that the lombok generated helpers are basically not there.   We are still trying to figure out what is happening here, but wanted to see if anyone has run into anything like this before.  It's unclear if upgrading to JobRunrPro somehow uncovers an issue with our configuration we did not know we had or if its something else.   When we drop back to JobRunr free, the project compiles and things are ok again.  

Any thoughts or assistance would be appreciated.
Joey

**Comment by rdehuyss** [comment]
Hi Joey,

Hmm, I don't understand as JobRunr and JobRunr Pro should be the same in regards to Lombok. I'll reach out to schedule a call with you so we can get to the bottom of this.

**Reply by jhanzel-be** [reply]
will do. Thank you
